src/lib/cost_model/cost_model_linear.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <map>
+#include <vector>
+
+#include "abstract_cost_model.hpp"
+#include "cost_feature.hpp"
+
+namespace opossum {
+
+/**
+ * CostModelLinear has different models for different kind of TableScans.
```

Comment:
Just a quick reminder to change the filenames and names in comments.

Reply:
dddone

Reply:
Resolved